File: SMI-SDK/.NET SDK/Library/gis/exception/IntializatonException.cs
namespace pt.sapo.gis.exception
{
    using System;
    /**
     *
     * @author GIS Team
     */
    public class IntializatonException : TraceException
    {

        public IntializatonException() { }

        public IntializatonException(String message) : base(message) { }

        public IntializatonException(String message, Exception cause) : base(message, cause) { }

    }
}

File: SMI-SDK/Java SDK/src/pt/sapo/gis/exception/IntializatonException.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package pt.sapo.gis.exception;

/**
 *
 * @author GIS Team
 */
public class IntializatonException extends TraceSDKException {

    public IntializatonException() {
    }

    public IntializatonException(String message) {
        super(message);
    }

    public IntializatonException(Throwable cause) {
        super(cause);
    }

    public IntializatonException(String message, Throwable cause) {
        super(message, cause);
    }
    
}
